README.md

```diff
@@ -19,19 +20,35 @@ The server exposes APIs that are consumed by the web app and the mobile app.
    ```
    SECRET = secure_server_secret
    MONGODB_URI = mongodb://localhost:27017/boldo
-
+   
```

Comment:
remove trailing whitespaces.
hint: use a formatter!

Reply:
Got it!